src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -120,8 +120,11 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
   private def startRunnableCalls(): State = {
     tryStartingRunnableCalls() match {
       case Success(entries) =>
-        if (entries.nonEmpty) tryStartingRunnableCalls()
-        goto(WorkflowRunning)
+        if (entries.nonEmpty) {
+          startRunnableCalls()
+        } else {
+          goto(WorkflowRunning)
```

Comment:
The reason why I changed this is because `tryStartingRunnableCalls` can return an error and we're currently ignoring that error on the inner invocation which causes the workflow to hang and not emit any errors.